app/scripts/reducers/api-version.js

```diff
@@ -0,0 +1,29 @@
+import {
+  API_VERSION,
+  API_VERSION_INFLIGHT,
+  API_VERSION_ERROR,
+  SET_API_VERSION_COMPAT_ERROR
+} from '../actions';
+
+export const initialState = {
+  api_version: '',
+  version_compat_error: false
```

Comment:
I think it's easier to follow if this property is named the same as the action property that will update it, so changing it to `isCompatible`